### DISCUSSION TITLE
How to define AltGr key combination with mode key.

### DISCUSSION THREAD

**Discussion by benibilme** [i3/i3#4986] [Q&A]
Hello, I am using Super key (Windows Key) as mode key. I am using a fairly customized keyboard and I switched AltGr key with Alt Key since I had to type a lot of non latin1 character.  Because of that I seldom use, alt key combinations in i3. Currently Super and switched AltGr keys are next to each other in perfect place.  I would like to bind additional actions by using current mode key and altgr key. How can I do that. It is easy to define with control, shift keys, but I have not seen it with AltGr key. 

Thank in advance.

**Comment by Cippo95** [comment]
I don't know with your keyboard and configuration but in my case AltGr is Mod5, give it a try.
